melodic/distribution.yaml

```diff
@@ -10457,6 +10457,21 @@ repositories:
       url: https://github.com/ros-controls/realtime_tools.git
       version: melodic-devel
     status: maintained
+  relative_positional_controller:
```

Comment:
This name seems pretty generic to me, and thus it might violate https://www.ros.org/reps/rep-0144.html#global-guidelines point 1 or 4.  This one is really subjective, as I could be convinced otherwise.  But given the fact that we are at the end of life for Melodic, I'm of the opinion to not put this in, and instead have the discussion about the name in https://github.com/ros/rosdistro/pull/37721 .


It's kind of subjective, as this is a generic package, but it isn't clearly used by large parts of the community. Thus I think this package would be a candidate for prefixing the name of the entity.

Reply:
Okay, I'm leaving this be for Melodic...